### ISSUE TITLE
Fails to find ios-device-lib.exe on windows

### ISSUE THREAD

**Issue by jcassidyav** [telerik/ios-device-lib#78]
running ``ns device`` on ns v8 gives the error:

```
Connected devices & emulators
Searching for devices...
Error: spawn C:\Users\user\AppData\Roaming\npm\node_modules\nativescript\node_modules\ios-device-lib\bin\win32\x64ios-device-lib ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:269:19)
    at onErrorNT (internal/child_process.js:465:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
```

it looks like the bug is in 
```ios-device-lib-stdio-handler.js```

```
return path.resolve(__dirname,`bin/win32/${os.arch()}ios-device-lib`
```
should be
```
return path.resolve(__dirname,`bin/win32/${os.arch()}/ios-device-lib`
```

See https://github.com/NativeScript/nativescript-cli/issues/5455

**Comment by MCorzo** [comment]
@jcassidyav thanks man, you saved my day 😀...

**Comment by pieveee** [comment]
@jcassidyav  Thank you so much!

**Comment by jcassidyav** [comment]
Closed with #79
